src/tokens/ERC1155/presets/lootbox/ERC1155Lootbox.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.19;
+
+import {IERC1155Lootbox, IERC1155LootboxFunctions} from "./IERC1155Lootbox.sol";
+import {ERC1155Items} from "@0xsequence/contracts-library/tokens/ERC1155/presets/items/ERC1155Items.sol";
+import {MerkleProof} from "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+import {IERC1155ItemsFunctions} from "@0xsequence/contracts-library/tokens/ERC1155/presets/items/IERC1155Items.sol";
+
+contract ERC1155Lootbox is ERC1155Items, IERC1155Lootbox {
+    bytes32 public merkleRoot;
+    uint256 public boxSupply;
+
+    mapping(address => uint256) private _commitments;
+    mapping(uint256 => bool) private _claimedIdxs;
+
+    constructor() ERC1155Items() {}
+
+    function setBoxContent(bytes32 _merkleRoot, uint256 _boxSupply) external onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
Let's use `bytes32 internal constant MINT_ADMIN_ROLE = keccak256("MINT_ADMIN_ROLE");` for this. Same as the Sale contracts.